memeid4s/src/main/scala/memeid4s/package.scala

```diff
@@ -1,177 +1,5 @@
-/*
- * Copyright 2019-2020 47 Degrees Open Source <http://47deg.com>
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
```

Comment:
There shouldn't be any problem since the [docs workflow](https://github.com/47degrees/memeid/blob/069be2355b4d75e673d66afee8ded24bdcfddd92/.github/workflows/docs.yml#L24) is going to auto-generate it (by using [this command](https://github.com/47degrees/memeid/blob/069be2355b4d75e673d66afee8ded24bdcfddd92/build.sbt#L7)), nonetheless, you're right, I shouldn't have removed it, here it is: 4279f6ed3c5006cff4a209d2da27e1a5d855f6a8